lib/src/revset.rs

```diff
@@ -831,6 +833,11 @@ static BUILTIN_FUNCTION_MAP: Lazy<HashMap<&'static str, RevsetFunction>> = Lazy:
             pattern,
         )))
     });
+    map.insert("signed", |_diagnostics, function, _context| {
+        function.expect_no_arguments()?;
+        let predicate = RevsetFilterPredicate::Signed;
+        Ok(RevsetExpression::filter(predicate))
+    });
```

Comment:
Can you add a few tests to `lib/tests/test_revset.rs`? I think some setup code can be copied from `test_signing.rs`.